imports/startup/server/accounts.js

```diff
@@ -32,6 +32,35 @@
  * admin roles on a fresh install (when number of users is zero)
  */
 
+//Set Meteor Admin Password ONLY if it has not been set.
+if(Meteor.settings.adminPassword==="password"){
+	
+	//Generating 8 Length Alphanumeric Password.
+	function passwordGen(len){
+	    var text = " ";
+	    var charset = "abcdefghijklmnopqrstuvwxyz0123456789";
+	    for( var i=0; i < len; i++ )
+		text += charset.charAt(Math.floor(Math.random() * charset.length));
+	    return text;
+	}
+	
+	//Applying Password to Meteor Admin User Settings and Logging to Server.
+	var randomPassword = passwordGen(8)
+	Meteor.settings = {
+		adminPassword: randomPassword,
+		private:{},
+		public : {
+			smtp: {
+				username: "postmaster%40sandbox5cb71a0119964fde80f91c415ef345a2.mailgun.org",
+				password: "b38c82be7ed0b4046bdc856547c655d3",
+				"server": "smtp.mailgun.org",
+				"port": 587
+			}
+		}
+	}
```

Comment:
Instead of changing all the settings in the function, we need to set only the adminPassword. 

One way to do this would be:
Can you see how you got the adminPassword from the file for if condition; Similarly, set other settings to what's in the file, instead of directly writing the values.